docs/docs/api/Dispatcher.md

```diff
@@ -1097,3 +1097,200 @@ or
   }
 }
 ```
+
+# Response Error Interceptor
```

Comment:
Can you add it under the [`built-in` interceptor section](https://github.com/nodejs/undici/blob/main/docs/docs/api/Dispatcher.md#pre-built-interceptors)?